sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/LocalKeyEncryptionKeyClientBuilder.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.cryptography;
+
+import com.azure.core.cryptography.AsyncKeyEncryptionKey;
+import com.azure.core.cryptography.KeyEncryptionKey;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.security.keyvault.keys.models.JsonWebKey;
+import reactor.core.publisher.Mono;
+
+/**
+ * This class provides a fluent builder API to help aid the configuration and instantiation of the {@link
+ * LocalKeyEncryptionKeyAsyncClient AsyncKeyEncryptionKey} and {@link LocalKeyEncryptionKeyClient KeyEncryptionKey},
+ * by calling {@link LocalKeyEncryptionKeyClientBuilder#buildAsyncKeyEncryptionKey(JsonWebKey)} and {@link
+ * LocalKeyEncryptionKeyClientBuilder#buildKeyEncryptionKey(JsonWebKey)} respectively.
+ * It constructs an instance of the desired client.
+ *
+ * <p> The minimal configuration options required by {@link LocalKeyEncryptionKeyClientBuilder} to
+ * build {@link LocalKeyEncryptionKeyAsyncClient} or {@link LocalKeyEncryptionKeyClient} is
+ * {@link JsonWebKey json web key}).</p>
+ *
+ * @see LocalKeyEncryptionKeyAsyncClient
+ * @see LocalKeyEncryptionKeyClient
+ */
+public final class LocalKeyEncryptionKeyClientBuilder {
```

Comment:
The build methods build KeyEncryptionKey and not the LocalKeyEncryptionKeyClient. Should this builder name be updated to reflect that?

Reply:
That's a good point,
But the builder name should just reflect the client it builds not the interfaces they implement.
The clients happen to implement KEK interfaces, which are used by Storage.
The build method names are there to be consistent with the KeyEncryptionKeyClientBuilder which implemtents KeyEncryptionKeyResolver.